### PR TITLE
Add project_services submodule

### PR DESCRIPTION
**Pull request by sosimon**
Adding a submodule to activate project APIs. Project Factory already does this, but having this submodule means it can be reused even for projects that were not created with Project Factory.

### CODE REVIEW

File: modules/project_services/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# project_services
+
+This optional module is used to enable project APIs in your project. By default,
+this module will enable the following APIs:
+- `compute.googleapis.com`
```
Comment (morgante) [inline modules/project_services/README.md]: No defaults please.

File: modules/project_services/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- `iam.googleapis.com`
+
+The list of APIs to be enabled can be customized with the `activate_apis`
+variable.
+
+Google Project Factory already does this [here](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/main.tf#L112), but this module allows you to enable APIs on projects that were not created with Project Factory.
```
Comment (morgante) [inline modules/project_services/README.md]: We probably don't need to link to Project Factory here.

File: modules/project_services/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Prerequisites
+
+1. Service account used to run Terraform has permissions to manage project APIs: 
+[`roles/serviceusage.serviceUsageAdmin`](https://cloud.google.com/iam/docs/understanding-roles#service-usage-roles) or `Owner`
```
Comment (morgante) [inline modules/project_services/README.md]: Let's be consistent.

```suggestion
[`roles/serviceusage.serviceUsageAdmin`](https://cloud.google.com/iam/docs/understanding-roles#service-usage-roles) or [`roles/owner`](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions) 
```

File: modules/project_services/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Usage
```
Comment (morgante) [inline modules/project_services/README.md]: People should not directly invoke the module. Instead include an example of using it inside a Terraform config.

File: modules/project_services/outputs.tf [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project_id" {
+  description = "The GCP project you want to enable APIs on"
+  value       = "${var.project_id}"
```
Comment (morgante) [inline modules/project_services/outputs.tf]: This needs to be pulled from `google_project_service` so the dependency graph works out.

```suggestion
  value       = "${google_project_service. project_services.project}"
```

**Comment by sosimon** [comment]
@morgante How do the changes look?

File: modules/project_services/README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Project API Activation
+
+This optional module is used to enable project APIs in your project. The list of
+APIs to be enabled is specified using the `activate_apis` variable.
+
+This module uses the [`google_project_service`](https://www.terraform.io/docs/providers/google/r/google_project_service.html)
+resource, which is  _non-authoritative_, as oppose to the [`google_project_services`](https://www.terraform.io/docs/providers/google/r/google_project_services.html)
```
Comment (morgante) [inline modules/project_services/README.md]:
```suggestion
resource, which is  _non-authoritative_, as opposed to the [`google_project_services`](https://www.terraform.io/docs/providers/google/r/google_project_services.html)
```

File: modules/project_services/README.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Prerequisites
+
+1. Service account used to run Terraform has permissions to manage project APIs: 
+[`roles/serviceusage.serviceUsageAdmin`](https://cloud.google.com/iam/docs/understanding-roles#service-usage-roles) or [`roles/owner`](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions)
+
+## Example
+
+See [examples/project_services](../../examples/project_services) for an example.
```
Comment (morgante) [inline modules/project_services/README.md]:
```suggestion
See [examples/project_services](./examples/project_services) for an example.
```

**Comment by morgante** [comment]
@sosimon Please resolve linting errors:

```
./modules/project_services/README.md:1. Service account used to run Terraform has permissions to manage project APIs: 
```

**Comment by sosimon** [comment]
@morgante Linting error fixed

**Pull request merged**